Makefile

```diff
@@ -395,6 +396,7 @@ prepare-extra-conda-environments:
 	$(MAKE) -C backend/python/huggingface
 	$(MAKE) -C backend/python/vall-e-x
 	$(MAKE) -C backend/python/exllama
+	$(MAKE) -C backend/python/backend_outlines
```

Comment:
this needs to be added also in the Dockerfile here: https://github.com/mudler/LocalAI/blob/991ecce00462db955934ec1293954b7d61c58ad2/Dockerfile#L153

Reply:
Done